distributed/shuffle/tests/test_shuffle.py

```diff
@@ -1010,41 +914,6 @@ async def test_closed_worker_between_repeats(c, s, w1, w2, w3):
     await clean_scheduler(s)
 
 
-@pytest.mark.slow
-@gen_cluster(client=True, nthreads=[("", 1)], Worker=Nanny)
-async def test_restart_cluster_between_repeats(c, s, a):
```

Comment:
this sounds like a rather generic test. Why remove it? Does this still provide value w/out the tombstone assertion?

Reply:
This test overlappe with `test_repeat` and checked a less generic case of the latter, i.e., we were only able to repeat shuffles if we restarted the cluster. `test_repeat` tests the more generic case of simply repeating them.